lib/matplotlib/testing/widgets.py

```diff
@@ -83,3 +83,31 @@ def do_event(tool, etype, button=1, xdata=0, ydata=0, key=None, step=1):
     event = mock_event(tool.ax, button, xdata, ydata, key, step)
     func = getattr(tool, etype)
     func(event)
+
+
+def click_and_move(tool, start, end, key=None):
+    """
+    Click at *start*, move to end*, and then relese the mouse.
+
+    Parameters
+    ----------
+    tool : matplotlib.widgets.RectangleSelector
+    start : [Number, Number]
```

Comment:
```suggestion
    start : (float, float)
```
would be our standard convention (https://matplotlib.org/stable/devel/documenting_mpl.html#parameter-type-descriptions).

Reply:
Thanks